bloom/generators/debian/templates/ament_cmake/copyright.em

```diff
@@ -1 +1,11 @@
-@(Copyright)
\ No newline at end of file
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: @(Name)
+Upstream-Contact: @(Maintainers.replace(', ', '\n '))
```

Comment:
For upstream-contact, when packaging in Debian, we typically use the issue tracker of the upstream project, see https://salsa.debian.org/science-team/ignition-utils/-/blob/master/debian/copyright#L3. Not sure if we can do it from $(Source) at least if we can detect github present. Could be that issues are not enabled for a given repository, not sure how to workaround that use case.

Reply:
I think we can actually use the `bugtracker` URL if present in the manifest: https://www.ros.org/reps/rep-0149.html#id18

Since that's optional, do you think it would be better to omit `Upstream-Contact` in the absence of a `bugtracker` URL, or fall back to using the maintainer list like this?

Reply:
> I think we can actually use the `bugtracker` URL if present in the manifest: https://www.ros.org/reps/rep-0149.html#id18

This is indeed much better, thanks.

Since the official spec says:
> May be free-form text, but by convention will usually be written as a list of RFC5322 addresses or URIs. 

Fallback to maintainer list, if we can access to the required attribute email, should be perfect. The [REP 149 establish that the email in required](https://www.ros.org/reps/rep-0149.html#maintainer-multiple-but-at-least-one).

Reply:
Because `Upstream-Contact` is listed as optional I'm in favor of omitting it when the bugtracker url is absent. I'd rather not encourage emailing maintainers directly.